README.md

```diff
@@ -12,12 +12,12 @@ Distinguishing features:
 * Coding style follows PSR-1, PSR-2, and Symfony2 conventions.
 * Auto-generate API documentation via [phpDocumentor 2.x](http://phpdoc.org/).
 
-[![Latest Stable Version](https://poser.pugx.org/instaclick/php-webdriver/v/stable.png)](https://packagist.org/packages/instaclick/php-webdriver)
-[![Total Downloads](https://poser.pugx.org/instaclick/php-webdriver/downloads.png)](https://packagist.org/packages/instaclick/php-webdriver)
+[![Latest Stable Version](https://poser.pugx.org/lullabot/php-webdriver/v/stable.png)](https://packagist.org/packages/lullabot/php-webdriver)
+[![Total Downloads](https://poser.pugx.org/lullabot/php-webdriver/downloads.png)](https://packagist.org/packages/lullabot/php-webdriver)
 
 ## Links
-* [Packagist](http://packagist.org/packages/instaclick/php-webdriver)
-* [Github](https://github.com/instaclick/php-webdriver)
+* [Packagist](http://packagist.org/packages/lullabot/php-webdriver)
+* [Github](https://github.com/lullabot/php-webdriver)
 * [W3C/WebDriver](https://github.com/w3c/webdriver)
 
 ## Notes
```

Comment:
This section can be deleted

Reply:
You mean the 
```
## Notes
* The *5.2.x* branch is no longer maintained. This branch features class names and file re-organization that follow PEAR/ZF1 conventions. Bug fixes and enhancements from the master branch likely won't be backported.
```
I think we should delete the 5.2 branch before we do that.